src/deployment-guides/tap.md

```diff
@@ -0,0 +1,799 @@
+* A container image registry with push and write access (such as Harbor, Docker Hub, or Azure Container Registry) is available. This will be used for application images and runtime dependencies.
+* Network access to [VMware's image registry](https://registry.tanzu.vmware.com) is available.
+<!-- /* cSpell:disable */ -->
+* DNS Records for components like Cloud Native Runtimes (knative) i.e. cnrs, Tanzu Learning Center, Tanzu Application Platform GUI etc.
+* A centralized git repository from GitHub, Gitlab or Azure DevOps for the Tanzu Application Platform GUI's software catalogs, along with a token allowing read access.
+* A Kubernetes workload cluster versions 1.20, 1.21, or 1.22 on Azure Kubernetes Service, Amazon Elastic Kubernetes Service, Google Kubernetes Engine, or [minikube](https://minikube.sigs.k8s.io/docs/start/) providers.
+<!-- /* cSpell:enable */ -->
+* Accept the End User License Agreements (EULAs).
+* The Kubernetes CLI, kubectl, v1.20, v1.21 or v1.22, installed and authenticated with administrator rights for your target cluster.
+
+Additional details concerning prerequisites may be found in Tanzu Application Platform [documentation](https://docs.vmware.com/en/Tanzu-Application-Platform/1.0/tap/GUID-prerequisites.html).
+
+
+## Overview of the Deployment Steps
+
+The following provides an overview of the steps necessary to deploy Tanzu Application Platform. Each step links to a detailed information section.
+
+1. [Setup Tanzu Application Platform Build cluster](#tap-build)
+1. [Setup Tanzu Application Platform Run cluster](#tap-run)
+1. [Setup Tanzu Application Platform View cluster](#tap-ui)
+1. [Setup Tanzu Application Platform Iterate cluster](#tap-full)
+1. [App Deployment](#tap-sample-app)
+
+
+## <a id="tap-build"> </a>Setup Tanzu Application Platform Build Cluster
+
+The Build Cluster is responsible for taking a developer's source code commits and applying a supply chain that will produce a container image and Kubernetes manifests for deploying on a Run Cluster
+
+### <a id="tanzu-essential"> </a> Step 1: Install Tanzu Cluster Essentials and Tanzu CLI
+
+Provide following user inputs into commands and execute them to install Tanzu Cluster Essentials and Tanzu CLI on your bootstrap/jumpbox machine. 
+
+* `TAP_TAP_WORKLOAD_CONTEXT` - the `kubeconfig` context of the workload cluster
+* `TANZU_NET_API_TOKEN` - an API token obtained from https://network.tanzu.vmware.com
+* `TANZU_NET_USER` - the Tanzu username
+* `TANZU_NET_PASSWORD` - the Tanzu user's password
+  
+```bash
+set -e
+
+export TAP_WORKLOAD_CONTEXT=<workload cluster context>
+export TANZU_NET_REGISTRY_ENDPOINT="registry.tanzu.vmware.com"
+export TANZU_NET_API_TOKEN=<tanzu refresh token>
+export TANZU_NET_USER=<tanzu username>
+export TANZU_NET_PASSWORD=<tanzu password>
+export TAP_INSTALL_BUNDLE=registry.tanzu.vmware.com/tanzu-cluster-essentials/cluster-essentials-bundle@sha256:82dfaf70656b54dcba0d4def85ccae1578ff27054e7533d08320244af7fb0343
```

Comment:
This variable is not referenced anywhere.

Reply:
@craigtracey - this is as per the tap documentations refer below steps and link - https://docs.vmware.com/en/Tanzu-Application-Platform/1.0/tap/GUID-install-tanzu-cli.html#Install%20Cluster%20Essentials%20for%20VMware%20Tanzu

Configure and run install.sh, which installs kapp-controller and secretgen-controller on your cluster:

export INSTALL_BUNDLE=registry.tanzu.vmware.com/tanzu-cluster-essentials/cluster-essentials-bundle@sha256:82dfaf70656b54dcba0d4def85ccae1578ff27054e7533d08320244af7fb0343
export INSTALL_REGISTRY_HOSTNAME=registry.tanzu.vmware.com
export INSTALL_REGISTRY_USERNAME=TANZU-NET-USER
export INSTALL_REGISTRY_PASSWORD=TANZU-NET-PASSWORD
cd $HOME/tanzu-cluster-essentials
./install.sh